### ISSUE TITLE
Add a watch to monitor the deployment of the join on the spoke cluster

### ISSUE THREAD

**Issue by itdove** [open-cluster-management-io/clusteradm#9]
Add the capability to watch the status and wait the completion of the spoke cluster agent installation.

**Comment by kim-fitness** [comment]
I think watch the pod `klusterlet-registration-agent` is in `Running` status is OK to continue to let user to do `accept` from hub cluster.

```
kubectl get pod -n open-cluster-management-agent -l "app=klusterlet-registration-agent"
```

**Comment by kim-fitness** [comment]
Hi @itdove,
This feature was implemented from PR #65.

The brief idea is watching the condition of CR klusterlet, and the `clusteradm join` command returns until the condition `Available` is `true` or timeouts in 300 seconds.

**Comment by kim-fitness** [comment]
Please @qiujian16 helps to close this issue as it already be implemented. Thanks.
